docs/csharp/language-reference/builtin-types/floating-point-numeric-types.md

```diff
@@ -89,15 +107,21 @@ For more information about implicit numeric conversions, see [Implicit Numeric C
 
 For more information about explicit numeric conversions, see [Explicit Numeric Conversions Table](../keywords/explicit-numeric-conversions-table.md).
 
+## C# language specification
+
+For more information, see the following sections of the [C# language specification](~/_csharplang/spec/introduction.md):
+
+- [Floating-point types](~/_csharplang/spec/types.md#floating-point-types)
+- [The decimal type](~/_csharplang/spec/types.md#the-decimal-type)
+- [Real literals](~/_csharplang/spec/lexical-structure.md#real-literals)
+
 ## See also
 
-- [C# Reference](../index.md)
+- [C# reference](../index.md)
 - [Integral types](integral-numeric-types.md)
 - [Built-in types table](../keywords/built-in-types-table.md)
 - [Numerics in .NET](../../../standard/numerics.md)
 - [Casting and Type Conversions](../../programming-guide/types/casting-and-type-conversions.md)
-- [Implicit Numeric Conversions Table](../keywords/implicit-numeric-conversions-table.md)
-- [Explicit Numeric Conversions Table](../keywords/explicit-numeric-conversions-table.md)
```

Comment:
@BillWagner I've removed those two links because they are explicitly mentioned in the section about conversions.
By the way, I see another consolidation opportunity here: merge those two articles into one called "Bulit-in numeric conversions". What do you think?